mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/ArenaAdapterService.kt

```diff
@@ -254,7 +249,6 @@ class ArenaAdapterService(
         arenaGjennomforing: ArenaTiltaksgjennomforingDbo,
         current: TiltaksgjennomforingAdminDto,
     ): Boolean {
-        val avslutningsstatus = tiltaksgjennomforinger.getAvslutningsstatus(current.id)
```

Comment:
Jeg fjernet denne og lager avslutningsstatus basert på status som gjort over i merge funksjonen. Var det noe spesiell grunn til at den ble hentet ut på egenhånd her? 🤔

Reply:
Eneste forskjell blir vel at IKKE_AVSLUTTET blir AVSLUTTET hvis slutt_dato er passert. Som er en potensiell evig loop hvis arena ikke oppdaterer til AVSLUTTET når vi skriver tilbake, men det må de jo gjøre og dette blir vel bare riktigere i så fall (fordi de får riktig status i Arena tidligere). 

Hmm litt usikker på hvilket case det her blir egentlig - at vi får tilbake en endring fra Arena etter at gjennomføringen er avsluttet (men ikke skrevet tilbake til Arena så den står som IKKE_AVSLUTTET der)... litt søkt?

Reply:
Dette ble lagt inn for å fikse opp i feilen som førte til at avsluttede gjennomføringer likevel ble vist til veiledere.

Når en gjennomføring blir satt til status `AVSLUTTET` i Arena så trigger det en endring som ender opp i duplikatsjekken her.
Når `current` blir hentet fra databasen så utleder vi samme status `AVSLUTTET` fordi man har passert `slutt_dato`, selv om gjennomføringen står som `IKKE_AVSLUTTET` i databasen.
Det førte til at vi ikke tok i mot endringen fra Arena fordi dbo'en ble ansett som en duplikat, men da slår det feil ut i spørringen for `VeilederTiltaksgjennomforing` som benytter avslsutningsstatus IKKE_AVSLUTTET til å avgjøre hva som blir hentet.

Hvordan dette blir med endringene du har her nå klarer jeg ikke helt å resonnere rundt akkurat nå :sweat_smile: